Modules/Filtering/Colormap/include/itkScalarToRGBColormapImageFilter.h

```diff
@@ -54,6 +54,7 @@ class ScalarToRGBColormapImageFilterEnums
 };
 #if !defined(ITK_LEGACY_REMOVE)
 using RGBColormapFilterEnum = ScalarToRGBColormapImageFilterEnums::RGBColormapFilter;
```

Comment:
Do we need line 56 if we have line 57?

Reply:
I am not sure which is needed for backward compatibility.  Error on the side of defining an unused name in the ITK_LEGACY_REMOVE section.